core/store/orm/orm_test.go

```diff
@@ -1357,3 +1357,32 @@ func TestJobs_ScopedInitiator(t *testing.T) {
 	expectation := []string{fmJob.ID.String(), twoInitrJob.ID.String()}
 	assert.ElementsMatch(t, expectation, actual)
 }
+
+func TestJobs_MoreThanBatchWithArchives(t *testing.T) {
+	store, cleanup := cltest.NewStore(t)
+	defer cleanup()
+	if store.Config.DatabaseURL() == "" {
+		return // skip if not using postgres
```

Comment:
Don't love this but sqlite blows up with 1000 jobs, `too many SQL variables`, and I don't think we have a pattern yet for testing specific databases.

Reply:
Considering sqlite support is still a thing and this would blow up in the same way for 1k+ job runs, should the batch be less than 1000?

Reply:
@jleeh Great suggestion, just updated it and the test is much cleaner now.